backend/src/packages/business/business.service.ts

```diff
@@ -136,11 +136,11 @@ class BusinessService implements IService {
     return await this.driverService.create({ payload, businessId });
   }
 
-  public updateDriver({
+  public async updateDriver({
     driverId,
     payload,
   }: DriverUpdatePayload): Promise<DriverCreateUpdateResponseDto> {
-    return this.driverService.update({
+    return await this.driverService.update({
```

Comment:
What's this await for?

Reply:
I suppose it's eslint rule to add await even async always returns promise